sympy/utilities/pytest.py

```diff
@@ -90,6 +91,43 @@ def raises(expectedException, code=None):
             raise TypeError(
                 'raises() expects a callable for the 2nd argument.')
 
+    def assert_raise_message(expectedException, err_msg, code=None):
```

Comment:
As an example, I think it might be better to consider `expectedException` to be either an instance of `Exception` or a tuple/list of `Exception` instances. So it might be better to be called `expectedExceptions` instead of `expectedException` (i.e. with an 's' at the end). Since I think sometimes we expect exception type to be a few possible values, instead of a single one.

WDYT?

Reply:
When do we need exceptions type to be multiple values?
Ok then one of the exceptions should be satisfied, and error message will also be tuple/list of strings.

Reply:
> Ok then one of the exceptions should be satisfied

Yes.

> error message will also be tuple/list of strings.

No. The err_msg should be a substring of the actual error message raised.

Reply:
This function will not be available when `USE_PYTEST` is `True`.

Reply:
This function will not be available when `USE_PYTEST` is `True`.

Reply:
Yes! It is inside `if not USE_PYTEST:` statement.

Reply:
@vishalg2235 It should be available in both cases.